examples/plot_grid_search_census.py

```diff
@@ -269,3 +269,7 @@ def classifier(X):
     show_plot=True,
 )
 # End model comparison
+
+
+#.. note::
+#   |edit| `View source <https://github.com/fairlearn/fairlearn/blob/main/examples/plot_grid_search_census.py>`to the code file or `open issue <https://github.com/fairlearn/fairlearn/issues/new?>`__.
```

Comment:
```suggestion
#   |edit| `View source <https://github.com/fairlearn/fairlearn/blob/main/examples/plot_grid_search_census.py>` or `open issue <https://github.com/fairlearn/fairlearn/issues/new?>`__.
```

Small suggestion for this file and others.

Did you build the docs to see if it worked as intended?

Reply:
Is there any way to put the note in rst or html file for the notebook examples? Putting it in .py file just makes it a part of the examples rather than making a note.